billsplit/src/utils/calculateTotalAmout.js

```diff
@@ -0,0 +1,27 @@
+
+export const calculateLogisticsTotal = bill => {
+  
+  if (bill.taxPercent.length === 0) {
+    let initial =
+      parseInt(bill.amountTotal) / parseInt(bill.numberOfPartyMembers);
+    let tipPerPerson = parseInt(bill.amountTotal) * (parseInt(bill.tipPercent) / 100);
+    let amountPerPerson = initial + tipPerPerson;
+    return amountPerPerson;
+  
+  } else if (bill.tipPercent.length === 0) {
+    let taxAmount = parseInt(bill.amountTotal) * (parseInt(bill.taxPercent) / 100);
+    let initial = parseInt(bill.amountTotal) + taxAmount
+    let amountPerPerson = (initial / (parseInt(bill.numberOfPartyMembers)))
```

Comment:
When you perform the per person calculation, you can use the `toFixed(2)` method to round the number down to just two decimal points. That way, we don't get an infinite tail of numbers after the decimal point.